server/repository/src/db_diesel/requisition_line/requisition_line.rs

```diff
@@ -17,11 +14,12 @@ use super::{
     RequisitionLineFilter, RequisitionLineRow,
 };
 
-pub type RequisitionLineJoin = (RequisitionLineRow, ItemLinkRow, RequisitionRow);
+type RequisitionLineJoin = (RequisitionLineRow, (ItemLinkRow, ItemRow), RequisitionRow);
```

Comment:
Refactored this to get the ItemRow and to not leak the ItemLinkRow out of this repository...